vm-virtio/src/console.rs

```diff
@@ -252,25 +283,61 @@ impl ConsoleInput {
         in_buffer.extend(input);
         let _ = self.input_evt.write(1);
     }
+
+    pub fn update_console_size(&self, cols: u16, rows: u16) {
+        self.config.lock().unwrap().update_console_size(cols, rows);
```

Comment:
Check that `VIRTIO_F_CONSOLE_F_SIZE` is in the acked_features before doing anything in this function.

Reply:
Done.